src/bot/scripts/card.js

```diff
@@ -50,1171 +52,1143 @@ const readDir = dirPath => {
-        ) {
-          return message.channel.send("Out of bounds");
-        }
+  const cards = cache.cachedData;
 
-        if (
-          commandValues.modifier == "u" &&
-          (commandValues.value > 70 || commandValues.value < 1)
-        ) {
-          return message.channel.send("out of bounds");
-        }
-        if (commandValues.modifier === undefined) {
-          commandValues.modifier = "l";
-        }
-        if (commandValues.value === undefined) {
-          commandValues.value = 1;
-        }
+  const split = message.content.split(" ");
 
-        if (commandValues.name === "%r %r") {
-          commandValues.name =
-            cards.data[Math.floor(Math.random() * cards.data.length)].name;
-          commandValues.modifier = "l";
-          commandValues.value = Math.floor(Math.random() * 7);
-        }
+  const splitWithoutCmd = split.slice(1);
 
-        commandValues.value = parseInt(commandValues.value);
+  //this is a fucking mess dont bother
+  let cardName = [];
 
-        let highestToDate = 0;
-        let highestCard = null;
+  let cardstats = [];
 
-        // card name - similarity
-        for (let card of cards.data) {
-          const lowerArray = card.name.toLowerCase();
+  let commandValues = [];
 
-          let sim = similarity(lowerArray, commandValues.name);
+  let cardValues = "";
 
-          if (sim > highestToDate) {
-            highestToDate = sim;
-            highestCard = card;
-          }
-        }
+  //gets name
+  for (let cmdWord of splitWithoutCmd) {
+    if (
+      cmdWord === "ff" ||
+      cmdWord === "art" ||
+      (cmdWord.startsWith("l") ||
+        cmdWord.startsWith("m") ||
+        cmdWord.startsWith("u")) &&
+      (!isNaN(parseInt(cmdWord.slice(1))) || cmdWord.slice(1) === "%r")
+    ) {
+      continue;
+    }
 
-        if (highestToDate < 0.4) {
-          return message.channel.send("Card Not Found");
-        }
+    cardName.push(cmdWord);
+  }
 
-        commandValues.matchedCards = highestCard;
-        commandValues.similarity = highestToDate;
+  //more code handling
+  cardName = cardName.join(" ").toLowerCase();
+  commandValues.name = cardName;
 
-        //const cars = ["ford-lemon",
-        //"lambo-range",
-        //"ferra-pple",
-        //"peug-rapes"];
+  //randomizes card
+  if (commandValues.name === "%r") {
+    commandValues.name =
+      cards.data[Math.floor(Math.random() * cards.data.length)].name;
+  }
 
-        const url = `https://sppd.feinwaru.com/backgrounds/${commandValues.matchedCards.image}.jpg`;
+  //checks it has level shit
+  for (let stats of splitWithoutCmd) {
+    if (
+      stats === "ff" ||
+      stats === "art" ||
+      (stats.startsWith("l") ||
+        stats.startsWith("m") ||
+        stats.startsWith("u")) &&
+      (!isNaN(parseInt(stats.slice(1))) || /%r$/g.test()) ||
+      stats === "u%r" ||
+      stats === "l%r"
+    ) {
+      cardstats.push(stats);
+    }
+  }
+  console.log(cardstats);
```

Comment:
errors are fine to log but this is just not necessary and would pollute the console output